client/src/modules/stock/inventories/templates/inventory.action.html

```diff
@@ -0,0 +1,17 @@
+<div class="ui-grid-cell-contents text-action" uib-dropdown dropdown-append-to-body uib-dropdown-toggle ng-if="!row.groupHeader">
```

Comment:
Can you make this `text-right`?  Most of our other actions buttons are right-aligned.